src/VCR/Videorecorder.php

```diff
@@ -228,13 +229,20 @@ public function handleRequest(Request $request)
             return $response;
         }
 
-        if (VCR::MODE_NONE === $this->config->getMode() || VCR::MODE_ONCE === $this->config->getMode() && $this->cassette->isNew() === false) {
+        if (VCR::MODE_NONE === $this->config->getMode()
+            || VCR::MODE_ONCE === $this->config->getMode()
+            && $this->cassette->isNew() === false
+        ) {
             throw new \LogicException(
-                "The request does not match a previously recorded request and the 'mode' is set to '{$this->config->getMode()}'. "
```

Comment:
Not sure how this `{$this->config->getMode()}` works. I've removed it when adding sprintf :see_no_evil: